docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -654,6 +654,79 @@ Finally, enable the necessary 802.1X and VOIP settings on each interface:
     lldp med network-policy add 1
     lldp med network-policy add 2
 
+GS950
+^^^^^
+
+PacketFence supports the GS950 switch using :
+
+* MAC Authentication (mac-only)
+* 802.1X (without fallback to MAC authentication)
+
+Global configuration
+++++++++++++++++++++
+
+First, ensure that the VLANs you want to assign are part of the VLAN database via the following page:
+
+Note that they only need to be tagged on the trunk and don't need any specific configuration for the dynamic VLAN assignment here.
+
+image::docs/images/allied-telesis-gs950/vlan-config.png[scaledwidth="100%",alt="VLAN configuration"]
+
+Next, configure the RADIUS server (Security->RADIUS):
+
+image::docs/images/allied-telesis-gs950/radius-server.png[scaledwidth="100%",alt="RADIUS configuration"]
+
+Next, configure an SNMP community (SNMP->Community Table)
+
+image::docs/images/allied-telesis-gs950/snmp-config.png[scaledwidth="100%",alt="SNMP configuration"]
+
+MAC authentication
+++++++++++++++++++
+
+Go in Security->Port Access Control, select the port you want to enable MAB on, and ensure you set:
+
+[options="compact"]
+* Authentication Mode: MAC Based
+* Port Control: Auto
+* Supplicant Mode: Single
+* VLAN Assignment: Enabled
+
+image::docs/images/allied-telesis-gs950/port-control-MAB.png[scaledwidth="100%",alt="MAB config"]
+
+802.1x
+++++++
+
+Go in Security->Port Access Control, select the port you want to enable MAB on, and ensure you set:
+
+[options="compact"]
+* Authentication Mode: MAC Based
+* Port Control: Auto
+* Supplicant Mode: Single
+* VLAN Assignment: Enabled
+
+image::docs/images/allied-telesis-gs950/port-control-802.1x.png[scaledwidth="100%",alt="802.1x configuration"]
+
+PacketFence configuration
++++++++++++++++++++++++++
+
+Ensure you configure at least:
+
+[options="compact"]
+* Type: Allied Telesis GS950
+* RADIUS secret: useStrongerSecret
+* SNMP Version: v2c
+* SNMP Community Read: private
+* SNMP Community Write: private
+
+If you are using MAC authentication on this switch, tyou must adjust the FreeRADIUS configuration so it transforms the EAP requests this switch sends into requests that PacketFence will interpret as MAC authentication. This is necessary because the 'MAC Based' mode of the switch still performs EAP requests.
```

Comment:
That's in the new updated English. It is a formal 'you'

Note that this doc was approved by @TheGreatZayme